.github/workflows/release-publish.yml

```diff
@@ -251,6 +251,7 @@ jobs:
       if: ${{ !env.ACT }}
       with:
         charts_dir: helm
+        charts_url: https://projectnessie.org/charts/
```

Comment:
Update the URLs in the release-notes as well?

Reply:
But I can have charts.projectnessie.org point to it, I guess?

Reply:
> But I can have charts.projectnessie.org point to it, I guess?
yep, I changed the URL here to also point to charts.projectnessie.org